app/javascript/mastodon/reducers/compose.js

```diff
@@ -155,6 +156,16 @@ const insertSuggestion = (state, position, token, completion, path) => {
   });
 };
 
+const ignoreSuggestion = (state, position, token) => {
+  return state.withMutations(map => {
+    map.set('suggestion_token', null);
+    map.set('suggestions', ImmutableList());
+    map.set('focusDate', new Date());
+    map.set('caretPosition', position + token.length + 1);
```

Comment:
This introduces a slight behavior inconsistency, with a space being added if the hashtag isn't a case-insensitive match, and no space being added if there is.

Reply:
Great catch. I'm not sure how to best to generate a space of it is necessary.

Reply:
Something like (untested):
```javascript
map.updateIn(path, oldText => `${oldText.slice(0, position + token.length)} ${oldText.slice(position + token.length)}`);
```

Reply:
Added that which worked.